pkgs/data/icons/volantes-cursors/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, inkscape
+, xcursorgen
+}:
+stdenv.mkDerivation {
+  pname = "volantes-cursors";
+  version = "2022-08-27";
```

Comment:
I'm not super sure what the standard is now, but I think this is fine.

Reply:
When there are no tags I've been told in the past to use the date of the commit. (And add -unstable if its unstable, but this package isn't unstable).